helm/kvm-operator/templates/vpa.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: {{ include "resource.default.name" . }}
+  namespace: {{ include "resource.default.namespace" . }}
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  resourcePolicy:
+    containerPolicies:
+    - containerName: {{ .Chart.Name }}
+      controlledValues: RequestsAndLimits
+      mode: Auto
+  targetRef:
```

Comment:
Does this ref not include a namespace?

Reply:
No it doesn't ([link](https://v1-16.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#crossversionobjectreference-v1-autoscaling)). I'm assuming it just looks for resources in the namespace where it's running.